ipaserver/install/replication.py

```diff
@@ -1028,7 +1029,12 @@ def check_repl_init(self, conn, agmtdn, start):
             refresh = entry.single_value.get('nsds5BeginReplicaRefresh')
             inprogress = entry.single_value.get('nsds5replicaUpdateInProgress')
             status = entry.single_value.get('nsds5ReplicaLastInitStatus')
+            json_status = entry.single_value.get('nsds5ReplicaLastInitStatusJSON')
             if not refresh: # done - check status
```

Comment:
Don't you need a call to json.loads(json_status)?

Reply:
Yup good catch, fixed in latest commit as well as pep8 fixes, but FYI that file was already filled with pep8 errors :-).

Reply:
Right, we got stricter as the project aged and rather than polluting the repo with a bunch of lint fixing changes we fix them as they come up, often, as in your case, when fixing unrelated but nearby code.